cibuild.cmd

```diff
@@ -79,14 +80,20 @@ if "%SYNC_XLF%"=="true" (
     set SYNC_XLF_ARGUMENT="/p:SyncXlf=true"
 )
 
+:: Full Framework MSBuild does not have the new built-in property MSBuildRuntimeType
+set RUNTIMETYPE_ARGUMENT=
+if "%TARGET%"=="Full" (
```

Comment:
I think we need to use MSBuild 14.0 to build the first phase which is what we do in our official builds.  Otherwise when I build locally the versions will be wrong for CoreCLR.  But at the moment we use the "downloaded" MSBuild for the first build.